### PR TITLE
[C#][Virtual Assistant] Add yaml file for C# Virtual Assistant sample functional tests

### PR DESCRIPTION
**Pull request by VictorGrycuk**
### Purpose
*What is the context of this pull request? Why is it being done?*
Adds a YAML file with the configuration for the C# Virtual Assistant sample functional tests pipeline

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
Adds a new YAML file

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

File: samples/csharp/assistants/virtual-assistant/VirtualAssistantSample.FunctionalTests/Nightly-Dotnet-VirtualAssistantToSkill.yml [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+         Write-Host "Resource group $(SkillBotName) exists, there is likely an error on the previous build. Deleting."
+          az group delete --name $(SkillBotName) --yes
+     }
+     else
+     {
+         Write-Host "Resource group $(SkillBotName) does not exist."
+     }
+     
+    workingDirectory: 'samples/csharp/assistants/virtual-assistant/VirtualAssistantSample/'
+  enabled: false
```
Comment (Batta32) [inline samples/csharp/assistants/virtual-assistant/VirtualAssistantSample.FunctionalTests/Nightly-Dotnet-VirtualAssistantToSkill.yml]: Is this okay? As I noticed that in the pipeline is disabled

image

Comment (VictorGrycuk) [reply samples/csharp/assistants/virtual-assistant/VirtualAssistantSample.FunctionalTests/Nightly-Dotnet-VirtualAssistantToSkill.yml]: It is configured like that in MS.
I will remove it from the YAML

Comment (Batta32) [reply samples/csharp/assistants/virtual-assistant/VirtualAssistantSample.FunctionalTests/Nightly-Dotnet-VirtualAssistantToSkill.yml]: I don't think that would be better to remove it. Leave it as it's like MS and it's ok!

**Comment by Batta32** [comment]
Can we rollback the remove of the task that you removed? If that's the behavior of MS, we should leave it as is

**Comment by DiegoCardozo94** [comment]
I recommend to @VictorGrycuk use this [YAML ](https://github.com/southworks/botframework-solutions/blob/88e1f9f1bb08be850c31582e66cc6dc35bf37b9a/templates/typescript/samples/sample-assistant/functional-tests/sample-assistant.yml) file as a guide. I detected some issues with the identation structure and spaces on each task. Also, check that some tasks has the `powershell` name, I suggest change to `pwsh` to use `powershell core`.

**Review by Batta32 — APPROVED** [review]

**Pull request merged**